backend/LeetWars.Core/LeetWars.Core.BLL/Interfaces/IPreferencesService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using LeetWars.Core.Common.DTO.UserPrefferences;
+
+namespace LeetWars.Core.BLL.Interfaces
+{
+    public interface IPreferencesService
```

Comment:
you can use User service for these methods

Reply:
moved